Cloud-CV/EvalAI
# #2291 OAuth Integration (Github)

> What is Added
- OAuth (github) using Django Allauth & Django RestAuth 

>Testing Steps
- Create an OAuth APP on Github, Redirect URL should be `<frontend-url>/auth/oauth-redirect/github/`.
- Set the FrontEnd URL in the environment variable FRONTEND_URL.
- Change client id in frontend (login page)  
- Follow Django AllAuth Documentation to add a site 

>What is left?
- I need to set client id for frontend in  env, I am unable to find where env files of the frontend are stored

I made some mistake while merging( #2353) with master so creating new MR, I have Incorporated all changes requested
